app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarMenuControllerTest.kt

```diff
@@ -562,25 +562,40 @@ class DefaultBrowserToolbarMenuControllerTest {
     }
 
     @Test
-    fun `WHEN sync sign in menu item is pressed AND account is signed out THEN navigate to sync sign in`() = runBlockingTest {
-        val item = ToolbarMenu.Item.SyncAccount(false)
-        val directions = BrowserFragmentDirections.actionGlobalTurnOnSync()
-
+    fun `GIVEN account exists and the user is signed in WHEN sync sign in menu item is pressed THEN navigate to account settings`() = runBlockingTest {
```

Comment:
```suggestion
    fun `GIVEN account is authenticated and the user is signed in WHEN sync sign in menu item is pressed THEN navigate to account settings`() = runBlockingTest {
```

Reply:
The below test would start with
> GIVEN account is authenticated and the user is not signed in ...
This doesn't sound right.

Reply:
```suggestion
    fun `GIVEN account is authenticated WHEN sync sign in menu item is pressed THEN navigate to account settings`() = runBlockingTest {
```